server/README.md

```diff
@@ -110,6 +110,19 @@ This demo implementation rejects login attempts by the demo user.
 
 This demo implementation logs a message to the demo channel whenever a user logs in.
 
+## [user\_hooks.go](user_hooks.go)
+
+### UserHasBeenCreated
+
+This demo implementation logs a message to demo channel when a new user is created.
```

Comment:
`when` --> `whenever` for consistency with how other hooks are described on the page.